documentation/how-to/tracking.en.md

```diff
@@ -18,8 +18,10 @@ QField allows you to create points, lines or polygons and track yourself in the
 
     You can choose between two options how the number of vertices will be recorded during the tracking session:
 
-    - A minimum time interval
-    - A minimum distance
+    - The `Time` tolerance records your position at a regular rate (e.g., every 30 seconds), which is ideal for consistent logs and saving battery.
+    - The `Distance` tolerance is more efficient, recording a point only after you move a minimum distance.
+    This keeps your track clean by ignoring movement when you are stationary.
+    To capture fine detail on steep or winding terrain, use a shorter distance, for straight routes, a longer distance is better.
```

Comment:
In the user interface this is called "Time requirement" and "Distance requirement". @nirvn might be convinced that tolerance is a better fit in terms of wording (@nirvn ?) but we should use a consistent terminology.

Reply:
I'm not convinced :wink:  This is not a tolerance, it is a requirement as indicated by both QFieldSync and QField.

<img width="585" height="221" alt="image" src="https://github.com/user-attachments/assets/69be1891-36a7-426a-a460-74e4153ec292" />

<img width="585" height="221" alt="image" src="https://github.com/user-attachments/assets/c8819c41-6f7a-43a4-b097-ebd2ce84fbbe" />

A tolerance indicates a "flexible" behavior, i.e. I'd read a time tolerance as "add at any time between 0 to 30 sec" :)